src/Maestro/tests/Scenarios/azdoflow-nonbatched-all-checks-successful.ps1

```diff
@@ -42,10 +42,10 @@ try {
     $targetRepoUri = Get-AzDO-RepoUri $targetRepoName
 
     Write-Host "Creating a test channel '$testChannelName'"
-    Darc-Add-Channel $testChannelName "test"
+    Darc-Add-Channel -channelName $testChannelName -classification "test"
 
     Write-Host "Adding a subscription from $sourceRepoName to $targetRepoName"
-    $subscriptionId = Darc-Add-Subscription --channel $testChannelName --source-repo $sourceRepoUri --target-repo $targetRepoUri --update-frequency none --target-branch $targetBranch --all-checks-passed --ignore-checks "'Work item linking'"
+    $subscriptionId = Darc-Add-Subscription @( "--channel", "$testChannelName", "--source-repo", "$sourceRepoUri", "--target-repo", "$targetRepoUri", "--update-frequency", "none", "--target-branch", "$targetBranch", "--all-checks-passed", "--ignore-checks", "Work item linking'" )
```

Comment:
```suggestion
    $subscriptionId = Darc-Add-Subscription @( "--channel", "$testChannelName", "--source-repo", "$sourceRepoUri", "--target-repo", "$targetRepoUri", "--update-frequency", "none", "--target-branch", "$targetBranch", "--all-checks-passed", "--ignore-checks", "Work item linking" )
```

Test was still failing because of trailing '